ts/usecases/rx_desc_nb.c

```diff
@@ -178,8 +177,16 @@ main(int argc, char *argv[])
     mbufs = TE_ALLOC(nb_packets * sizeof(*mbufs));
     CHECK_NOT_NULL(mbufs);
     TEST_STEP("Receive packets on @p iut_port");
-    received = test_rx_burst_with_retries(iut_rpcs, iut_port->if_index,
-                                          TEST_RXQ, mbufs, nb_packets, nb_rxd);
+
+    /*
+     * There are NIC controllers that can queue more received packets then
+     * the number of RX descriptors, e.g. QEMU network devices.
+     *
+     * So that, the only number of packets read at once should be validated.
+     */
+    received = rpc_rte_eth_rx_burst(iut_rpcs, iut_port->if_index, TEST_RXQ,
+                                    mbufs, MIN(nb_packets, UINT16_MAX));
```

Comment:
The gist of the test is to check a number of Rx descriptors and it can't be done using Rx with retries because of extra queueing byQEMU  virtio. Respecting the above limitation is out-of-scope of the patch and there are a number of other places in the TS which do not respect it.